openlibrary/plugins/openlibrary/js/graphs/index.js

```diff
@@ -0,0 +1,37 @@
+import { loadGraph, loadEditionsGraph } from './plot';
+import options from './options.js';
+
+export function plotAdminGraphs() {
+    if ($('#editgraph').length) {
+        loadGraph('editgraph', {}, 'edit(s) on');
+        loadGraph('membergraph', {}, 'new members(s) on');
+    }
+    if ($('#works_minigraph').length) {
+        loadGraph('works_minigraph', {}, ' works on ');
```

Comment:
I'm too happy with these functions but they increase my confidence in not breaking things as they align closely with how the code currently works.

I think in future these could use classes and data attributes on the graph themselves